src/mozxchannel/android/importer.py

```diff
@@ -26,6 +27,10 @@ def main():
     subprocess.run(
         ["git", "-C", args.dest, "commit", "-m", "Import l10n."], check=True
     )
+    if args.verbose:
+        subprocess.run(
+            ["git", "-C", args.dest, "diff", "master"], check=True
```

Comment:
what is with the hardcoded `master` here?

Reply:
I'll drop that for now, it also errors for one repo and I don't know why.